### DISCUSSION TITLE
"useDropZone" is not exported or doesn't exist

### DISCUSSION THREAD

**Discussion by onurusluca** [vueuse/vueuse#1655] [Q&A]
[useDropZone](https://vueuse.org/core/usedropzone/) doesn't exist in vueuse/core.
I can use other helpers but this one looks broken. It gives this error:
`Module '@vueuse/core' has no exported member 'useDropZone'`

I looked at the file and yes there isn't anything about useDropZone. For example there is useEyeDropper.

`import { useDropZone } from '@vueuse/core'
`

What am I doing wrong?

**Comment by onurusluca** [comment]
@Julien-Martin Sorry for bothering. Is there something wrong with useDropZone?

**Reply by Julien-Martin** [reply]
Hi :wave: @onurusluca,

I just try on a fresh install with `"@vueuse/core": "^8.6.0"` and it works as expected.

Did you upgrade `vueuse` to the last version ?

**Reply by Julien-Martin** [reply]
If you try it from the playground, it seems like the playground use an older version of `vueuse` :thinking:

**Reply by onurusluca** [reply]
@Julien-Martin Hi! Thanks a lot for the reply. Yes I updated to the last version. It seems to not have useDropZone .
I will downgrade for now then. Yes I realized that too and thought it was because of the last update.

**Reply by Julien-Martin** [reply]
Hum that it's strange, can you do a reproductible environnement on stackblitz/codesandbox/github ? :man_shrugging:
